translations/hu-HU.json

```
@@ -0,0 +1,2970 @@
+      "gemstone_slot_type.OFFENSIVE": ["Offensive"],
+      "gemstone_slot_type.DEFENSIVE": ["Defensive"],
+      "gemstone_slot_type.MINING": ["Mining"],
+      "gemstone_slot_type.UNIVERSAL": ["Universal"],
+      "faction.BARBARIANS": ["Barbarians"],
+      "faction.MAGES": ["Mages"]
+    }
+  },
+  "skyblock leaderboard":{
+    "name": "ranglista",
+    "description": "Kijelzi a SkyCrypt ranglistáit (sky.shiiyu.moe)",
+    "options": [
+      {
+        "name": "ranglista",
+        "description": "A kijelzendő ranglista."
+      },
+      {
+        "name": "játékos",
+        "description": "A ranglistán keresendő játékos neve."
+      }
+    ]
+  },
+  "skyblock wiki": {
+    "name": "wiki",
+    "description": "Keres egy cikket a SkyBlock Wiki-ről.",
+    "options": [
+      {
+        "name": "keresés",
+        "description": "A keresőszó amire keresni akarsz."
+      }
+    ],
+    "responses": {
+      "title": ["Hypixel SkyBlock Wiki"],
+      "noResults": ["Nincs találat a **{search}** keresőszóra."],
+      "results": ["Találatok a **{search}**: keresőszóra\n{results}"],
+      "wikiURL": ["A wiki megtalálható a [wiki.hypixel.net](https://wiki.hypixel.net/) oldalon."]
+    }
+  },
+  "skyblock time": {
+    "name": "idő",
+    "description": "Kijelzi a jelenlegi időt a SkyBlock-ban.",
+    "options": [],
+    "responses": {
+      "time": ["Jelenleg {time} óra van {date} napján a Hypixel SkyBlock-ban."]
+    }
+  },
+  "skyblock events": {
+    "name": "eventek",
+    "description": "Kezeli a SkyBlock eventek értesítéseit."
+  },
+  "skyblock-events": {
+    "name": "skyblock-eventek",
+    "description": "Kezeli a SkyBlock eventek értesítéseit."
+  },
+  "skyblock-events config": {
+    "name": "konfiguráció",
+    "description": "Konfigurálja a szerveren a SkyBlock eventek értesítéseit."
+  },
+  "skyblock-events config view": {
+    "name": "megtekintés",
+    "description": "Kijelzi a szerver SkyBlock eventek értesítéseinek a konfigurációját.",
+    "options": [],
+    "responses": {
+      "title": ["SkyBlock eventek értesítései"],
+      "fieldTemplate": [
+        "Enabled: {enabled}\n",
+        "Alerts: {alerts}\n",
+        "Ping Role: {ping}\n"
+      ],
+      "allAlerts": "Összes",
+      "noAlerts": "Semelyik"
+    }
+  },
+  "skyblock-events config toggle": {
+    "name": "kapcsoló",
+    "description": "Be- és kikapcsolja az eventek értesítéseit.",
+    "options": [
+      {
+        "name": "event",
+        "description": "A kapcsolandó event.",
+        "choices": [
+          "New Year",
+          "Spooky Festival",
+          "Jerry's Workshop nyílik",
+          "Season of Jerry",
+          "Mayor Election véget ér",
+          "Travelling Zoo",
+          "Fear Mongerer",
+          "Dark Auction",
+          "Mayor Election kezdődik",
+          "Bank Interest"
+        ]
+      }
+    ],
+    "responses": {
+      "enabled": [" **{event}** bekapcsolva."],
+      "disabled": [" **{event}** kikapcsolva."]
```

Spacing, unless this was also done in the English translation
```suggestion
      "enabled": ["**{event}** bekapcsolva."],
      "disabled": ["**{event}** kikapcsolva."]
```